packages/kbn-test/src/es/test_es_cluster.ts

```diff
@@ -35,6 +35,13 @@ interface TestClusterFactoryOptions {
    * - archive data folder: zip -r my_archive.zip data
    * */
   dataArchive?: string;
+  /**
+   * Name of the directory for the node which will be created in `.es/`,
+   * e.g. Providing `my-test-cluster` will create `.es/my-test-cluster`.
+   *
+   * When not provided, the current ES version is used as the directory name.
+   */
+  installDirName?: string;
```

Comment:
This was needed to prevent each node from running into write locks when trying to write to the same `data` directory.